docs/tutorial/security.md

```diff
@@ -249,15 +249,15 @@ prevent the use of Node primitives, `contextIsolation` must also be used.
 
 ### Why?
 
-Context isolation allows each the scripts on running in the renderer to make
-changes to its JavaScript environment without worrying about conflicting with
-the scripts in the Electron API or the preload script.
+Context isolation allows scripts on running in the renderer to make changes to
+their JavaScript environment without worrying about conflicting with the scripts
+in the Electron API or the preload script.
```

Comment:
Maybe add something about "or accidentally leaking global APIs that could allow for privilege escalation"